addons/addon-base-raas/packages/base-raas-services/lib/environment/environment-mount-service.js

```diff
@@ -138,30 +147,44 @@ class EnvironmentMountService extends Service {
               },
             },
           };
+          // Read Permission
           let getStatement = {
             Sid: getSid,
             Effect: 'Allow',
             Principal: { AWS: [] },
             Action: ['s3:GetObject'],
             Resource: [`arn:aws:s3:::${s3BucketName}/${prefix}*`],
           };
+          // Write Permission
+          let putStatement = {
+            Sid: putSid,
+            Effect: 'Allow',
+            Principal: { AWS: [] },
+            Action: ['s3:PutObject'],
+            Resource: [`arn:aws:s3:::${s3BucketName}/${prefix}*`],
+          };
```

Comment:
Nit: Probably put a comment here stating why PutObjectAcl is not required. It seems very counter intuitive that the IAM policy requires it but the bucket policy doesn't.

Reply:
Will do, good call.